frontend/public/components/utils/file-input.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+
+export class FileInput extends React.Component<fileInputProps, fileInputState> {
```

Comment:
👍 nice to see this added with so little code

Reply:
yeah, even the drag&Drop is pretty small :+1: